test/parallel/test-tls-enable-trace.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto) common.skip('missing crypto');
+const fixtures = require('../common/fixtures');
+
+// Test enableTrace: option for TLS.
+
+const assert = require('assert');
+const { fork } = require('child_process');
+
+if (process.env.CHILD)
+  return test();
+
+const child = fork(__filename, {
+  silent: true,
+  env: { CHILD: true },
```

Comment:
> ```
> not ok 1932 parallel/test-tls-enable-trace
>   ---
>   duration_ms: 0.370
>   severity: fail
>   exitcode: 1
>   stack: |-
>     Could not load program node:
>     rtld: 0712-001 Symbol _ZTTNSt7__cxx1119basic_ostringstreamIcSt11char_traitsIcESaIcEEE was referenced
>           from module node(), but a runtime definition
>     	    of the symbol was not found.
> ```
> 
> @nodejs/platform-aix Any idea why https://github.com/sam-github/node/blob/tls-trace/src/tls_wrap.cc#L915-L931 or something in https://github.com/sam-github/node/blob/tls-trace/deps/openssl/openssl/ssl/t1_trce.c would pulling in an undefined C++ iostreams symbol?


Please extend `env` rather than replace it to preserve `LIBPATH`.